include/LLVMSPIRVExtensions.inc

```diff
@@ -37,3 +37,4 @@ EXT(SPV_INTEL_loop_fuse)
 EXT(SPV_INTEL_long_constant_composite)
 EXT(SPV_INTEL_optnone)
 EXT(SPV_INTEL_memory_access_aliasing)
+EXT(SPV_INTEL_invocation_pipelining_attributes)
```

Comment:
The extension is 'SPV_INTEL_fpga_invocation_pipelining_attributes' in the spec.

Reply:
Good catch, thanks!

Reply:
And I need to align the appropriate code with EXT checking

Reply:
For sure :)